config/custom-environment-variables.yaml

```diff
@@ -64,8 +64,9 @@ datastore:
         prefix: DATASTORE_SEQUELIZE_PREFIX
 
 executor:
-    plugin: EXECUTOR_PLUGIN
```

Comment:
I might be wrong, but I thought what @stjohnjohnson  meant was to set `EXECUTOR_PLUGIN` to something like https://github.com/screwdriver-cd/screwdriver/blob/master/config/custom-environment-variables.yaml#L81-L83